### DISCUSSION TITLE
Integrate the RD++ algorithm

### DISCUSSION THREAD

**Discussion by letmejoin** [open-edge-platform/anomalib#2183] [Feature Requests]
### What is the motivation for this task?

The paper proposes the RD++ approach for anomaly detection by enriching feature compactness and suppressing anomalous signals through a multi-task learning design. For the feature compactness task, RD++ introduces the self-supervised optimal transport method. For the anomalous signal suppression task, RD++ simulates pseudo-abnormal samples with simplex noise and minimizes the reconstruction loss.
    RD++ achieves a new state-of-the-art benchmark on the challenging MVTec dataset for both anomaly detection and localization. More importantly, when compared to recent SOTA methods, RD++ runs 6.x times faster than PatchCore and 2.x times faster than CFA, while introducing a negligible latency compared to RD.

### Describe the solution you'd like

https://github.com/tientrandinh/Revisiting-Reverse-Distillation/tree/main

**Comment by blaz-r** [comment]
If you want to implement this I belive that it would be a great addition, however I'm not sure if anyone is already working on this. @samet-akcay

**Comment by samet-akcay** [comment]
No one from the team is working on this at the moment

**Comment by haimat** [comment]
That would really be a great addition!
